up-l3/usubscription/v3/README.adoc

```diff
@@ -18,139 +18,615 @@ SPDX-FileType: DOCUMENTATION
+--
+
+[.specitem,oft-sid="req~usubscription-state-remote-unsub-response-positive~1",oft-needs="impl,utest"]
+--
+When receiving a positive response to the forwarded unsubscribe request, the local uSubscription service *MUST* set the topic state to `UNSUBSCRIBED`.
+--
+
+[.specitem,oft-sid="req~usubscription-state-remote-unsub-response-negative~1",oft-needs="impl,utest"]
+--
+When receiving a negative response to the forwarded unsubscribe request, the local uSubscription service *MUST* set the topic state to `UNSUBSCRIBED_PENDING`.
+--
 
 == Subscription Change Notifications
 
-The uSubscription service notifies observers when there is a change in a subscription states by publishing notification events to the topic `/core.usubscription/3/subscriptions#Update`.  Below are the specific nuances about the subscription change notification based on the observer type.
+=== Default notification topic
+
+As has been described in the xref:usubscription-local-remote-subscriptions[remote subscriptions section] above, uProtocol dispatchers (`uStreamers`) are expected to intelligently forward messages to remote systems in cases where a remote uEntity (and thus a remote uSubscription service instance) made a subscription request to our local uProtocol network. 
+
+[.specitem,oft-sid="dsn~usubscription-change-notification~1",oft-needs="impl,utest"]
+--
+To support this use case, uSubscription service *MUST* make all subscription changes available as event messages, published on a topic specifically defined for this purpose.
+--
 
-===  Subscribers
+[.specitem,oft-sid="dsn~usubscription-change-notification-type~1",oft-needs="impl,utest"]
+--
+Subscription change messages *MUST* be of type https://github.com/eclipse-uprotocol/up-spec/blob/da5ca97d3a7541d2fcd52ed010bc3bcca92e46cb/up-core-api/uprotocol/core/usubscription/v3/usubscription.proto#L302[Update].
+--
 
-Subscribers are automatically registered to receive subscription change notifications for topics they have subscribed too through the Subscribe() API.
+[#usubscription-default-notification-topic]
+[.specitem,oft-sid="dsn~usubscription-change-notification-topic~1",oft-needs="impl,utest"]
+--
+uSubscription service *MUST* publish change notification messages to the topic
+ `/core.usubscription/3/subscriptions#Update`.
+--
 
-* Subscribers *MUST NOT* call the `RegisterForNotifications(`) as they are not permitted to receive notifications for subscription changes by other subscribers
-* Subscription change notification *MUST* be sent whenever the `SubscriptionState` changes for said subscriber (ex, `SUBSCRIBE_PENDING` → `SUBSCRIBED`)
+=== Custom notification topics
 
-=== Publishers
+In addition to publishing subscription change notifications on the xref:usubscription-default-notification-topic[default notification channel], uEntities may want to receive subscription change notifications on a topic of their choice. 
```

Comment:
This is not true. Publishers manually register to receive subscription change notifications on the same notification topic (the one you mentioned above). the `RegisterForNotifications()` and `UnregisterForNotifications()` APIs are used to explicitly "opt-in" to receiving subscription notifications or not but that notification is always delivered to the same subscription change notification topic.  that is defined in [usubscription.proto](https://github.com/eclipse-uprotocol/up-spec/blob/6b8c38b378daf9f4b8623b45e692e67517007a40/up-core-api/uprotocol/core/usubscription/v3/usubscription.proto#L38).

Reply:
Ok explain this to me like I'm five years old: what exactly does this do, from the point of view of usubscription service? Because the service is publishing sub-change notifications to that topic anyways, for any subscription it is tracking - that is my understanding. 
So what would usubscription do on top of that, when a client does `RegisterForNotifications()`? Isn't it a client-side decision whether to opt-in to listening on that default topic?

Reply:
Ah - so the misunderstanding was that this mechanism actually has nothing to do with pubsub communication, but is saying "hey, direct-send me those updates, please!".
I was in pubsub country in my head for all of this... makes more sense now.

I assume it is allowed for clients to subscribe to topics that are not published by them?